lib/provider/bpmn/parts/EventProps.js

```diff
@@ -66,7 +66,7 @@ module.exports = function(group, element, bpmnFactory, elementRegistry, translat
         var showErrorCodeVariable = isCatchingErrorEvent,
             showErrorMessageVariable = isCatchingErrorEvent;
 
-        error(group, element, bpmnFactory, errorEventDefinition, showErrorCodeVariable, showErrorMessageVariable);
+        error(group, element, bpmnFactory, errorEventDefinition, showErrorCodeVariable, showErrorMessageVariable, translate);
```

Comment:
```suggestion
        error(group, element, bpmnFactory, errorEventDefinition, showErrorCodeVariable, showErrorMessageVariable,
          translate);
```
Let's try to not exceed 100 characters per line.